subsys/net/l2/openthread/Kconfig

```diff
@@ -141,6 +141,16 @@ config OPENTHREAD_RADIO_WORKQUEUE_STACK_SIZE
 	int "OpenThread radio transmit workqueue stack size"
 	default 512
 
+config OPENTHREAD_FP_GUARD_EXTRA_STACK_MARGIN
+	int "Margin by which the OpenThread stacks are increased with FP"
+	help
+	  If enabled the OpenThread stacks will be increased by a margin
+	  which could be taken by the increased MPU STACK GUARD when
+	  the usage of the FP registers takes place.
+	  For details check the z_arm_thread_stack_info_adjust call.
+	default 96 if FPU && FPU_SHARING && MPU_STACK_GUARD
+	default 0
```

Comment:
This looks weird as usually the help section is the last option in the kconfig files. Doing it like this seems to work but it would be better if the various kconfig options look similar, so please move the default options before the help option.

Reply:
Thanks for a very quick review. We discussed internally and came up with a better solution. Could you revisit?